docs/api-testing/contract-testing.md

```diff
@@ -79,15 +84,13 @@ To test the API consumer (client) side:
 
   Be sure to use the same OpenAPI spec used to test the API producer side. This will activate the contract testing functionality and bind a series of endpoints with a Sauce Labs API Testing project.
 
-  The execution will create a [log](/api-testing/project-dashboard/#test-logs) in your Project's [**Dashboard**](/api-testing/project-dashboard/), specific to contract testing. Here, you'll be able to see the results of both the API Producer and API Consumer tests. The report document for the contract test will detail how the request and response appear during the transaction and the nature of any test failures, if applicable.
-
   :::tip
   Use the [`--validate-request`](/api-testing/mocking/#validate-request) switch to ensure your requests are compliant with the schema.
   :::
 
-2. Next, make an API call to one of the endpoints in your spec file that Piestry has generated a mock for. To do so, you can use any HTTP Client &#8212; or &#8212; execute the same endpoint you used to test the API Producer. This will validate both sides of the contract. Once executed, you'll see the mock response if it matches the response contract. At this point, you also generated a log that is recorded in the dashboard.
+3. Next, make an API call to one of the endpoints in your spec file that Piestry has generated a mock for. To do so, you can use any HTTP Client &#8212; or &#8212; execute the same endpoint (from your **Snapshots**) you used to [test the API Producer side](#testing-the-api-producer-side). This will validate both sides of the contract. Once executed, you'll see the mock response if it matches the response contract. At this point, you also generated a log that is recorded in the dashboard.
```

Comment:
No, it does not make sense

Reply:
Hi @LoryApiFortress , this change came from Patrick -- I've sent you both a Slack message

Reply:
Hi @LoryApiFortress , this version you were looking at is Outdated (has the label). You can see the updated version with that change applied by clicking on the files tab --> https://github.com/saucelabs/sauce-docs/pull/1113/files